lib-auth add description for scope NONE #350 (#351)
enonic/doc-xp: docs/api/lib-auth.adoc

| idProvider | string | Name of id provider where the user is stored. If not specified it will try all available id providers, in alphabetical order.
| skipAuth | boolean | Skip authentication.  Default is `false` if not specified.
| sessionTimeout | number | Session timeout (in seconds). By default, the value of session.timeout from `com.enonic.xp.web.jetty.cfg`

> | scope image:xp-730.svg[XP 7.3.0,opts=inline] | string | Defines the scope of the login. Valid values are `SESSION`, `REQUEST` and `NONE` (starting from version image:xp-780.svg[XP 7.8.0,opts=inline]). When scope is set to `SESSION` the login is persistent. If scope is set to `REQUEST` the login is only valid for the current request. If scope is set to `NONE` that allows to check if a user has a valid username and password without logging into the system. Default is `SESSION` if not specified.

Reviewer: ```suggestion
| scope image:xp-730.svg[XP 7.3.0,opts=inline] | string | Defines the scope of the login. Valid values are `SESSION`, `REQUEST` and `NONE` (starting from version image:xp-780.svg[XP 7.8.0,opts=inline]). When scope is set to `SESSION` the login is persistent. If scope is set to `REQUEST` the login is only valid for the current request. Scope `NONE` allows to check correctness of username and password without logging into the system. Default is `SESSION` if not specified.
```
